src/api/public/apidocs-new/paths/build_project_repository_architecture_package_file.yaml

```diff
@@ -106,3 +106,70 @@ put:
             details: "404 project 'home:Admin' has no repository '4'"
   tags:
     - Build
+
+delete:
+  summary: This endpoint deletes an existing repository artifact
+  description: |
+    Given a project, repository and architecture delete the given file in the repository.
```

Comment:
The update operation done with PUT can only be done as ADMIN. Could you also check if this also is required with the DELETE action?

Reply:
Confirmed. Only Admin can do this